rskj-core/src/main/java/co/rsk/config/BridgeConstants.java

```diff
@@ -86,6 +86,10 @@ public abstract class BridgeConstants {
 
     protected int numberOfBlocksBetweenPegouts;
 
+    protected long estimatedPegoutTxIndexBtcActivationHeight;
```

Comment:
```suggestion
    protected long btcHeightWhenPegoutTxIndexActivates; 
```
Just to keep it consistent with the other constant here `btcHeightWhenBlockIndexActivates`

Reply:
Done